web_src/fomantic/build/semantic.js

```diff
@@ -2118,7 +2118,7 @@ $.fn.dimmer = function(parameters) {
         moduleNamespace = 'module-' + namespace,
         moduleSelector  = $allModules.selector || '',
 
-        clickEvent      = ('ontouchstart' in document.documentElement)
+        clickEvent = "click", unstableClickEvent = ('ontouchstart' in document.documentElement)
```

Comment:
Isn't this `unstableClickEvent` a unused variable? Why not replace the value with just `"click"`?

Reply:
That's the best and simplest I can do by a sed command. Leave an unused variable there won't cause any problem.